lib/active_admin/resource/attributes.rb

```diff
@@ -0,0 +1,31 @@
+module ActiveAdmin
+
+  class Resource
+    module Attributes
+
+      def default_attributes
+        return {} unless resource_class.respond_to?(:columns)
```

Comment:
@deivid-rodriguez , don't remember context, maybe for activemodel compatibility or something... looks like it is better to remove this since 
I don't see any test for this to be present